mitodl/mitxpro
# #428 #447 #448 Multiple Fixes

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #428 #447 #448 

#### What's this PR do?
Closes aforementioned tickets

#### How should this be manually tested?
Go to home page and product page:
- All carousels should have autoplay turned off
- All carousels should have infinite scroll turned off
- Courseware carousel now has clickable cards (navigates to the content page)
- Courseware carousel cards now do not spawn a new tab
- Courseware carousel card title is now a link
- Courseware carousel card should now have increased vertical padding around the "view detail" link
- "More Dates" section now links to the checkout page with the appropriate content if the course run has a product associated with it
